docs/content/1.guide/6.assets.md

```diff
@@ -82,27 +89,14 @@ export default defineNuxtConfig({
 ```
 ::
 
-They can be addressed by the key `assets/my_directory/`.
-
-### Examples
-
-**Example:** Retrieving a json data from default `assets` directory:
-
-```js
-export default defineEventHandler(async () => {
-  const data = await useStorage().getItem(`assets/server/data.json`)
-  return data
-})
-```
-
-**Example:** Retrieving a html file from a custom `assets` directory:
+You could want to add a directory with html templates for example.
 
 ::code-group
 ```js [nitro.config.ts]
 export default defineNitroConfig({
   serverAssets: [{
     baseName: 'templates',
-    dir: './templates' // Relative to `srcDir` (`server/` for nuxt)
+    dir: './templates' // Relative to `srcDir` (`.` for nitro)
```

Comment:
i think this is mainly nitro docs so we should keep using note for nuxt

Reply:
I'm not sure to understand because there is still the part for nuxt just after. Otherwise, it's a bit redundant with next lines.

https://github.com/unjs/nitro/pull/2088/files/2b01fe99ef695b35426d2ec14b31009ab5f9f00d#diff-7d93f9e22cae88517c8b35815afbd6e8905707011913c417b21663063eb8f9d1R103-R111

Reply:
```suggestion
    dir: './templates'
```